apps/daimo-mobile/src/action/key.ts

```diff
@@ -10,7 +10,7 @@ import { defaultEnclaveKeyName } from "../model/account";
 function getKeySecurityMessage(hwSecLevel: ExpoEnclave.HardwareSecurityLevel) {
   switch (hwSecLevel) {
     case "SOFTWARE":
-      return "🔒  Key generated in Secure Enclave";
+      return "🔒  Key generated in software";
```

Comment:
minor but i'd either remove the key logo here or add it on `Key generated in trusted hardware` below for consistency

Reply:
removed, makes sense